packages/docs/src/playground/core/lib/runtime/main.runtime.ts

```diff
@@ -77,12 +77,13 @@ export default class Runtime {
     this.terminal = new Terminal(this._instance);
     this.xterm = xterm;
 
-    const terminalReady = Promise.withResolvers<void>();
+    const terminalReady = Promise.withResolvers<undefined>();
```

Comment:
void is correct here. Not sure why ts-eslint complains about it, but we can silence it